Source/Scene/Cesium3DTileset.js

```diff
@@ -879,6 +879,17 @@ import TileOrientedBoundingBox from './TileOrientedBoundingBox.js';
     }
 
     defineProperties(Cesium3DTileset.prototype, {
+        /**
+         * NOTE: This getter exists so that `Picking.js` can differentiate between
+         *       PrimitiveCollection and Cesium3DTileset objects without inflating
+         *       the size of the module via `instanceof Cesium3DTileset`
+         */
+        isCesium3DTileset : {
```

Comment:
My mistake for not noticing this in the initial review - the getter should be marked as `@private`.